src/Routing/routes.js

```diff
@@ -25,6 +25,10 @@ function RouteConfiq() {
         <Route
           path="/lc-admin/dashboard"
           element={<Layout Component={Dashboard} />}
+        />
```

Comment:
make sure all the components in same manner, pehle sare admin ke aayenge then user ke start karne